flyway/migrations/V2__Create_species_table.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE Species (
+    SpeciesID uniqueidentifier DEFAULT NEWSEQUENTIALID() PRIMARY KEY,
```

Comment:
The naming convention for SpeciesID is incorrect. Must have PK_ to shows it is a primary key